### DISCUSSION TITLE
How to edit PO translation files

### DISCUSSION THREAD

**Discussion by sayyad84** [OrchardCMS/OrchardCore#12649] [Q&A] [answered]
Please explain how to change PO translation files. After running the program, the changes made in PO files will be lost.

**Comment by hishamco** [comment, marked as answer]
Of course, the proper way to change the translation via Crowdin, which will reflect to `OrchardCore` repo. We push a NuGet package on every OC release.

In case you want to edit the translations in your local without losing them, simply you can comment the translation task in the target file

https://github.com/OrchardCMS/OrchardCore/blob/c2a6a2667c22eb017734c175bba929d4665815ab/src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/OrchardCore.Application.Cms.Core.Targets.targets#L30-L34

**Reply by hishamco** [reply]
This way the build will skip overriding your newly modified translations
